From f48e9694a6ef3260b7271b242cdade7c600a4ebe Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Wed, 21 Nov 2018 15:31:13 +0100
Subject: [PATCH] Require Windows Vista

caching_sha2_password requires CryptImportPublicKeyInfoEx2, which requires at least Windows Vista
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b812ab5..1924d2d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -246,6 +246,8 @@ IF(WIN32)
   SET(HAVE_THREADS 1)
   ADD_DEFINITIONS(-DHAVE_DLOPEN)
   ADD_DEFINITIONS(-D_CRT_SECURE_NO_WARNINGS)
+  ADD_DEFINITIONS(-DNTDDI_VERSION=0x06000000)
+  ADD_DEFINITIONS(-D_WIN32_WINNT=0x0600)
   IF(MSVC)
      SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /wd4996" )
   ENDIF()
